workers/unity/Packages/com.improbable.gdk.tools/.CodeGenerator/GdkCodeGenerator/Templates/UnityCommandManagerGenerator.tt

```diff
@@ -184,54 +189,10 @@ namespace <#= qualifiedNamespace #>
                 return true;
             }
 
-            private void AddRequest(CommandRequestOp op)
```

Comment:
What happened to this?

Reply:
That just gets swapped in directly from the diff. Ultimately this is all probably going to be removed and replaced directly by the diff stuff.